### ISSUE TITLE
getUser is blowing up in server handler

### ISSUE THREAD

**Issue by mikenicklas** [nuxt-modules/kinde#123]
I have a brand new Nuxt 3 app and a `/api/me` nuxt server handler which will attach a few extra things stored in a database. I'm getting this error: `ERROR  [nitro] [unhandledRejection] Compact JWS must be a string or Uint8Array` and this is the code:

```
export default defineEventHandler(async (event) => {
  const kinde = event.context.kinde
  const user = await kinde.getUser()

  return {
    doesnt: 'matter'
  }
})
```

Full stack trace:
```
 ERROR  [nitro] [unhandledRejection] Compact JWS must be a string or Uint8Array

  at compactVerify (node_modules/jose/dist/node/esm/jws/compact/verify.js:9:15)
  at jwtVerify (node_modules/jose/dist/node/esm/jwt/verify.js:5:28)
  at node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:142:45
  at step (node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:32:23)
  at Object.next (node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:13:53)
  at fulfilled (node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:4:58)
```

FWIW, `await kinde.isAuthenticated()` returns true, so I'm not sure how it would know if someone is authenticated or not if some cookie/jwt/etc. wasn't being passed and successfully verified.

My .env is very barebones and I basically followed your quick start when I selected Nuxt as my existing application:

```
NUXT_KINDE_PASSWORD=nope
NUXT_KINDE_CLIENT_ID=nope
NUXT_KINDE_CLIENT_SECRET=nope
NUXT_KINDE_AUTH_DOMAIN=https://mydomain.kinde.com
NUXT_KINDE_REDIRECT_URL=http://localhost:3000/api/callback
NUXT_KINDE_LOGOUT_REDIRECT_URL=http://localhost:3000
NUXT_KINDE_POST_LOGIN_REDIRECT_URL=http://localhost:3000/dashboard
```

Do you have any suggestions?

**Comment by sjkey** [comment]
You could use `getUserProfile()` it should get you decent amount of user properties

**Comment by DanielRivers** [comment]
This appears to be an issue with validation of the idToken stored, let me see if I can reproduce locally.

**Comment by DanielRivers** [comment]
This seems to be unreproducible on the nuxt starter kit. 

![image](https://github.com/user-attachments/assets/d4b39db7-3095-469a-a533-d223a4b90f84)

Could you share your package.json file please?

**Comment by DanielRivers** [comment]
I have noticed that `getUser` does return undefined, however `getUserProfile` will work its just not a performant as it requests from the server.

**Comment by DanielRivers** [comment]
I have been able to reproduce this issue

**Comment by mikenicklas** [comment]
@DanielRivers Thank you for fixing this. Are you planning to cut a new patch version for this? Looks like the latest released version is 0.1.10 so this is still erroring out on new installs.
